pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -521,7 +518,11 @@ func parseQuery(model *simplejson.Json) (*CloudWatchQuery, error) {
 
 func formatAlias(query *CloudWatchQuery, stat string, dimensions map[string]string) string {
 	if len(query.Id) > 0 && len(query.Expression) > 0 {
-		return query.Id
+		if len(query.Alias) > 0 {
```

Comment:
I suggest to change default alias of the case `len(query.Id) > 0` and `len(query.Expression) == 0` and `len(query.Alias) == 0`.

This is a suggested change. Please check :-)
https://github.com/grafana/grafana/commit/c5556954739b7a2e10f4eeaa548f9d6f5f67943f

Reply:
Same comment as above - I don't think we need a default value as when there is no alias defined, we are displaying the query as it is and handling the case on line 524. By assigning a default value, we are adding a complexity just for the sake of assigning a default value to a variable. By default in go, string values are empty string so we handle that in the else condition on line 524. @bergquist - Any suggestions?

Reply:
@mtanda Any comments?